huaweicloud/services/apig/resource_huaweicloud_apig_acl_policy_associate.go

```diff
@@ -61,6 +70,80 @@ func ResourceAclPolicyAssociate() *schema.Resource {
 	}
 }
 
+func aclPolicyBindingRefreshFunc(client *golangsdk.ServiceClient, instanceId, policyId string,
```

Comment:
What this method actually does is unbind.

Reply:
when we are binding APIs to a ACL policy, we will check whether include any API which is not bound again and again, and this operation is during the creation, so I names it as the `aclPolicyBindingRefreshFunc`.